### ISSUE TITLE
Style and accessibility Improvements

### ISSUE THREAD

**Issue by adeleke5140** [Tanza3D/reddark#22]
# Issue 
1. The webapp tends to fill up the whole space on a webpage, making the text stretch all the way from the left to the right.
2. The sound toggle button cannot be toggled with a keyboard. You need to hover and click on it with a mouse. 

## Why?
1. The app centered in the browser will aid readability as you wouldn't have to move your head to read text in a single column.

It would be great if it was nicely centered on the screen; like this.
Before:
<img width="1216" alt="Screenshot 2023-06-10 at 17 56 21" src="https://github.com/Tanza3D/reddark/assets/32024147/563cd0fa-1652-41fd-b7a4-acb43aa2727a">

When centered properly:
<img width="1199" alt="Screenshot 2023-06-10 at 18 02 34" src="https://github.com/Tanza3D/reddark/assets/32024147/5a50c086-d8ed-4127-bbe0-9bbf81c1416c">

2. Improving the accessibility of the button would make sure that keyboard users can navigate the web app as well.

**Comment by Tanza3D** [comment]
it's wide because personally i like to fit as many as i can onto my screen, on a 1080p screen it still gets centered:
![image](https://github.com/Tanza3D/reddark/assets/33783503/42edf154-7b26-425f-aeaa-a0b8f2129a2a)

agreed that it's an issue that keyboard users can't navigate, will have to look into ways to fix that

**Comment by Tanza3D** [comment]
^ i could probably base the width of the page on the screen width if that's something which would be good (using vw)

**Comment by filchermcurr** [comment]
For what it's worth, I like it the way it is. Information density is one reason I have such a large monitor. I feel like anybody who needs a smaller workspace for readability could just resize the window.

The 'proper' centering is way too narrow, wastes a ton of space, and would result in 3,000 years of scrolling.

**Comment by adeleke5140** [comment]
> For what it's worth, I like it the way it is. Information density is one reason I have such a large monitor. I feel like anybody who needs a smaller workspace for readability could just resize the window.
> 
> 
> 
> The 'proper' centering is way too narrow, wastes a ton of space, and would result in 3,000 years of scrolling.

I slightly disagree. There is data to support that reading is more comfortable when you don't have to move your head from the end of one column to the next. You might have a large monitor but I'd argue that that's just a small percentage of people. 

The scrolling in this case is also just going to be vertical. No horizontal scrolling required. Mouse users on mobile devices already do need to scroll and desktop users would have no issues with it.

@Tanza3D Regarding the button accessibility, I can help with that.

**Comment by adeleke5140** [comment]
> it's wide because personally i like to fit as many as i can onto my screen, on a 1080p screen it still gets centered:
> 
> ![image](https://github.com/Tanza3D/reddark/assets/33783503/42edf154-7b26-425f-aeaa-a0b8f2129a2a)
> 
> 
> 
> agreed that it's an issue that keyboard users can't navigate, will have to look into ways to fix that

Yes I saw this effect but on a 27inch monitor. The width I proposed can be adjusted to make sure there is little/more white space.

**Comment by Tanza3D** [comment]
it's not reading. it's a grid of text. no one reads this like text.

**Comment by adeleke5140** [comment]
> it's not reading. it's a grid of text. no one reads this like text.

Understandable. 

Would you be taking contributions for accessibility improvements?

**Comment by adeleke5140** [comment]
Just saw that there's a PR already for the accessibility improvements.
